nova/tests/unit/virt/vmwareapi/fake.py

```diff
@@ -726,14 +744,18 @@ def __init__(self, name="ha-host", connected=True, ds_ref=None,
                  maintenance_mode=False):
         super(HostSystem, self).__init__("host")
         self.set("name", name)
-        if _db_content.get("HostNetworkSystem", None) is None:
+
+        if not _db_content.get("HostNetworkSystem", None):
             create_host_network_system()
-        if not _get_object_refs('HostStorageSystem'):
+
+        if not _db_content.get("HostStorageSystem", None):
```

Comment:
Yes and no. I changed the interface from `_get_object_refs` (which was externally used) to return an iterator instead of the list, so needed a way to test if there are any instances. Since it is internal, and a one-liner,  I didn't bother to create a function for it